nats/replica_client.go

```diff
@@ -0,0 +1,402 @@
+package nats
```

Comment:
This doesn't seem to be imported or usable by the `cmd/litestream` binary. Am I missing something?

Reply:
I had hooked it up (i thought) in `cmd/litestream/replicate.go` around line 125